tests/remap/andypugh/toplevel.py

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/python
```

Comment:
/usr/bin/python is not installed on my machine. There is https://packages.debian.org/sid/main/python-is-python3 to help with some extra convenience - but, nah, or?